test/load-tests/api.yml

```diff
@@ -0,0 +1,19 @@
+config:
+  target: "https://localhost:8080"
+  phases:
+    - duration: 10
+      name: 'Low load'
+      arrivalRate: 3
+    # - duration: 10
+    #   name: 'Medium load'
+    #   arrivalRate: 20
+    # - duration: 10
+    #   name: 'High load'
+    #   arrivalRate: 50
+  processor: './load-test-utils.js'
+      
+scenarios:
+  - flow:
+    - get:
+        url: "/users"
+        beforeRequest: 'setAuthHeaders'
```

Comment:
![image](https://user-images.githubusercontent.com/16080021/28204567-ca4959d6-687e-11e7-8747-15ad340dc4da.png) from the docs.